src/components/Charts/PriceChart.tsx

```diff
@@ -1,17 +1,38 @@
+import { AxisBottom, TickFormatter } from '@visx/axis'
 import { localPoint } from '@visx/event'
 import { EventType } from '@visx/event/lib/types'
 import { GlyphCircle } from '@visx/glyph'
 import { Group } from '@visx/group'
 import { Line, LinePath } from '@visx/shape'
-import { bisect, scaleLinear } from 'd3'
+import { bisect, curveBasis, NumberValue, scaleLinear } from 'd3'
 import { radius } from 'd3-curve-circlecorners'
+import { useActiveLocale } from 'hooks/useActiveLocale'
 import useTheme from 'hooks/useTheme'
+import { TimePeriod } from 'hooks/useTopTokens'
 import { useCallback, useState } from 'react'
 import { ArrowDownRight, ArrowUpRight } from 'react-feather'
 import styled from 'styled-components/macro'
+import {
+  dayHourFormatter,
+  hourFormatter,
+  monthDayFormatter,
+  monthFormatter,
+  monthYearDayFormatter,
+  monthYearFormatter,
+  weekFormatter,
+} from 'utils/formatChartTimes'
 
 import data from './data.json'
 
+const TIME_DISPLAYS: Record<string, TimePeriod> = {
```

Comment:
i think you want the other way around. `Record<TimePeriod, string>` so it's type safe